defaults/main.yml

```diff
@@ -1,13 +1,12 @@
 ---
 
-# Options are standard or json
-sansible_nginx_access_log_format: json
+sansible_nginx_access_log_format: json  # either 'standard' or 'json'
 sansible_nginx_gzip: "on"
 sansible_nginx_gzip_comp_level: 2
 sansible_nginx_gzip_min_length: 1100
 sansible_nginx_gzip_static: "on"
 sansible_nginx_gzip_types: |
-    text/plain text/css application/x-javascript \
+    text/plain text/css application/x-javascript
```

Comment:
Does this definitely produce the desired string without the backslash?

Reply:
No, I forgot to change the `|` to `>`